### ISSUE TITLE
icnga2's old bundled nsclient++ crashes if server has no pagefile

### ISSUE THREAD

**Issue by Sec42** [Icinga/icinga2#6755]
The bundled version of NSClient++ crashes while trying to monitor "pagefile-windows" on servers with no page file.

## Expected Behavior
Don't crash :-)

## Current Behavior

```
INF: Detected crash...
INF: Running: "C:\Program Files\NSClient++\reporter.exe" "archive" "C:\Windows\SERVIC~2\NETWOR~1\AppData\Local\Temp\\939152e3-aa47-4f1b-88e8-c92a11d22d2b.dmp" "NSClient++" "0.5.0.62" "2016-09-14" "C:\Program Files\NSClient++/crash-dumps"

ERR: Failed to check service state

INF: Running: "C:\Program Files\NSClient++\reporter.exe" "restart" "NSCP"
```

## Possible Solution
The bundled NSClient++ seems to lag several versions behind.
A manual upgrade to 0.5.2.35 seems to fix the issue

## Your Environment

* Version used (`icinga2 --version`): r2.10.1-1

**Comment by stevie-sy** [comment]
Maybe you should put this issue to this channel: https://github.com/mickem/nscp/issues ;-)

**Comment by Sec42** [comment]
NSClient++ fixed this a long time ago. See https://github.com/mickem/nscp/issues/341
The issue is that icinga2 bundles an old version of NSClient++ with the agent

**Comment by stevie-sy** [comment]
ok thanks for this info. I think with the new title is more clear, what's the Problem is

**Comment by Crunsher** [comment]
In that case we'll have to update the bundled nsclient. Will require some testing with the snapshot packages once we get to it.

**Comment by dnsmichi** [comment]
I've been doing that with #5633 and this involves changes to the REST API / check_nscp_api. I'd rather wait up until the new REST API is fully implemented and released.

**Comment by lippserd** [comment]
Closed because we have to wait for #5633.
